src/Main.elm

```diff
@@ -24,7 +26,7 @@ init =
 
 -- UPDATE
 
-type Msg = Increment | Decrement
+type Msg = Increment | Decrement | Reset
 
 update : Msg -> Model -> Model
 update msg model =
```

Comment:
How would you read these lines? (29 and 30)

Reply:
See: https://guide.elm-lang.org/types/custom_types.html

I'd read like this:

The Msg type has three **variants**. A Msg can be `Increment`, `Decrement`, or `Reset`